### ISSUE TITLE
2+ line list item with anchor not properly aligned

### ISSUE THREAD

**Issue by gustavohenke** [angular/material#7233]
I have a list of 2 line items, and when an anchor is applied to them, the content gets top-aligned,
instead of centrally aligned.

Screenshot:
![](http://i.imgur.com/4eeyJIh.png)

[Live demo](http://codepen.io/gustavohenke/pen/BjgOJE)

Browsers tested:
- Chrome
- Firefox
- MS Edge

**Comment by gustavohenke** [comment]
This seems to be happening because `.md-list-item-inner` (created when we have an anchor) does not get the proper `min-height` when inside an `.md-2-line`.

*probable* fix:

```css
md-list-item.md-2-line .md-list-item-inner {
  min-height: 72px;
}
```

**Comment by ElzioJunior** [comment]
+1

**Comment by gustavohenke** [comment]
The same problem seems to happen with 3 line items.
I updated the the demo to include them as well.

**Comment by DevVersion** [comment]
Just want to stick that into that issue - the right fix for the issue is the missing addition for my previous commit bb5c105. See the PR for the addition #7060

**Comment by DevVersion** [comment]
Fixed with  SHA  db763bc
